tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -19,13 +19,49 @@ module.exports = {
         .click('@coreMenuOpenButton')
         .waitForElementVisible('@coreMenu')
         .waitForAnimationToFinish()
+    },
+    clickOnMarkAsRead: function () {
+      return this.waitForElementVisible('@ocTopbar')
+        .initAjaxCounters()
+        .waitForOutstandingAjaxCalls()
+        .waitForElementVisible('@notificationPanel', 35000)
```

Comment:
why this non-default waiting time?

Reply:
While testing it happened very often that the notification fetch happened before the notification got send. Then we have to wait for 30 seconds until they get fetched again.

Reply:
what about reloading the page to query immediately?

Reply:
@individual-it Changed it but no idea if this works.... Couldn't test as the acceptance tests don't work correctly right now